qiskit_experiments/database_service/db_analysis_result.py

```diff
@@ -417,6 +422,30 @@ def auto_save(self, save_val: bool) -> None:
             self.save()
         self._auto_save = save_val
 
+    @staticmethod
+    def _format_db_display(value):
+        if isinstance(value, (int, bool)):
+            return value
+        if isinstance(value, float):
+            if math.isfinite(value):
+                return value
+            else:
+                return serialize_safe_float(value)["__value__"]
+
+        # Values that we can store by string conversion
+        if isinstance(value, complex):
+            # Convert to string and truncate to 6 significant digits
+            value = "{:.8g}".format(value)
+        elif isinstance(value, (list, np.ndarray)) and len(value) < 10:
```

Comment:
This serializes `[1,2,3,4,1e-10j,1]` as `None`. Also, why not return a partial list with `"..."` as the last element if it's long?

Reply:
If you think there is value in only showing part of a long list in db can change it to that

Reply:
I feel like there would be, but maybe experimentalists can chime in here.